test/e2e_test.sh

```diff
@@ -71,7 +71,7 @@ for repo in rekor fulcio; do
     ${docker_compose} up -d
     echo -n "waiting up to 60 sec for system to start"
     count=0
-    until [ $(${docker_compose} ps | grep -c "(healthy)") == 3 ];
+    until $(${docker_compose} ps --format '{{json .}}' | jq -s -e '.[] | .Health == "healthy" or .Health == ""');
```

Comment:
It looks like checking for an empty string doesn't only mean the healthcheck is undefined. I just tested this command locally with a failing deployment (docker compose up for Fulcio where the database and Trillian fails to start). The containers are stuck in a loop restarting, not showing as unhealthy, with `.Health == ""`.

Reply:
Looking into this, it may be better to only check for the readiness, or health, of rekor-server and fulcio-server.

Some related work https://github.com/sigstore/rekor/pull/2397